core/pkg/launch/job_builder.go

```diff
@@ -702,12 +702,25 @@ func (j *JobBuilder) HandleUseArtifactRecord(record *service.Record) {
 func (j *JobBuilder) MakeJobMetadata(output *data_types.TypeRepresentation) (string, error) {
 	metadata := make(map[string]interface{})
 	input_types := make(map[string]interface{})
+	input_schemas := make(map[string]interface{})
 	if len(j.configFiles) > 0 {
 		files := make(map[string]interface{})
+		file_schemas := make(map[string]interface{})
 		for _, configFile := range j.configFiles {
 			files[configFile.relpath] = j.generateConfigFileSchema(configFile)
+			if configFile.inputSchema != nil {
+				var inputSchemaMap map[string]interface{}
+				err := json.Unmarshal([]byte(*configFile.inputSchema), &inputSchemaMap)
+				if err != nil {
```

Comment:
question about the data that could be included in `configFile.inputSchema`: 
what types are you expecting can be in the schema, want to verify that only standard json supported types are included here.

cc: @timoffex  as you are working through the json pr

Reply:
The input schema will be a valid [JSON Schema](https://json-schema.org/understanding-json-schema/reference), so it will only contain JSON-able types.